cmd/dockerd/notify_systemd_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build !nosystemd,!freebsd
```

Comment:
The document itself is a bit out of date, but would be good to add this to https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/project/PACKAGERS.md#docker_buildtags

I see we used `no_` for the `no_lcow` build tag, so perhaps for consistency it should be `no_systemd`

Reply:
That said; in situations where the daemon is not running as a systemd service, this code would not be doing anything, so I'm wondering what problem you're trying to solve https://github.com/moby/moby/blob/9a82a9a8eae52e3f46c6d7f1a2b073a9bd9b1c4f/vendor/github.com/coreos/go-systemd/v22/daemon/sdnotify.go#L56-L65

Reply:
this patch is for reducing unnecessary code, environments where there will never ever be any systemd and binaries should be as small as possible. (and as few LOC as possible to audit).